scripts/run_collator.sh

```diff
@@ -50,7 +50,7 @@ bootnode () {
     echo "$bootnode"
 }
 
-args+=( "--" "--wasm-execution=compiled" "--chain=${chain}" "--bootnodes=$(bootnode "$alice" "$alice_rpc_port")" "--bootnodes=$(bootnode "$bob" "$bob_rpc_port")" )
+args+=( "--" "--wasm-execution=compiled" "--pool-type=fork-aware" "--chain=${chain}" "--bootnodes=$(bootnode "$alice" "$alice_rpc_port")" "--bootnodes=$(bootnode "$bob" "$bob_rpc_port")" )
```

Comment:
@aramikm is there a way to make the fork aware pool default instead of needing to set it?

Reply:
I think it should be possible. I'll look into it.

Reply:
Since the `Configuration` is part of the code we depend on, one thing we can do is just check for the  SealingMode. and default it to ForkAware pool 
`if SealingMode != Instant `

Reply:
That sounds good. I'm also ok with just forcing the pool one way in Instant sealing mode. It's a special mode, so its ok to ignore that setting (as there are no forks in instant mode anyway).

In the end, we'll get that all collators and such upgrade, they default to the fork aware pool and avoid having to ask people to enable it when we need it.